specs/lifespan.rst

```diff
@@ -2,7 +2,7 @@
 Lifespan Protocol
 =================
 
-**Version**: 1.0 (2018-09-06)
+**Version**: 2.0 (2019-03-04)
```

Comment:
@andrewgodwin Just a note/reminder we need to add the `spec_version` to the scope.